lms/views/admin/application_instance.py

```diff
@@ -165,17 +139,7 @@ def upgrade_instance(self, lti_registration, consumer_key):
 
         # Set the LTI1.3 values
         application_instance.lti_registration = lti_registration
-        application_instance.deployment_id = self.request.params[
-            "deployment_id"
-        ].strip()
-
-        # And upgrade any other values present in the request
-        if email := self.request.params["email"].strip():
-            application_instance.requesters_email = email
-
-        if lms_url := self.request.params["lms_url"].strip():
-            application_instance.lms_url = lms_url
-
+        application_instance.deployment_id = deployment_id
         try:
             # Flush here to find if we are making a duplicate in the process of
             # upgrading
```

Comment:
The layout of this class is a little chaotic. It seems `upgrade_instance` should be next to all these `<verb>_instance` methods?